modules/FormModel.js

```diff
@@ -538,23 +852,33 @@
      * @method __setupListeners
      */
     __setupListeners: function() {
-      _.each(this.__modelConfigs, function(modelConfig) {
-        if (modelConfig.fields) {
-          _.each(modelConfig.fields, function(field) {
-            this.listenToModelField(modelConfig.model, field);
-          }, this);
+      var formModel = this;
+      _.each(formModel.getMappings(), function(config, alias) {
+        if (config.computed) {
+          var modelConfigs = formModel.__getComputedModelConfigs(alias);
+          _.each(modelConfigs, function(modelConfig) {
+            var model = modelConfig.model;
+            if (modelConfig.fields) {
+              _.each(modelConfig.fields, function(field) {
+                formModel.__listenToComputedValuesDependency(model, field, alias);
+              });
+            } else {
+              formModel.__listenToComputedValuesDependency(model, undefined, alias);
```

Comment:
then __listenToModelField wouldn't parallel this function anymore... hmm... Does formModel.__listenToComputedValuesDependency(model, '', alias); look better? it's still a falsey value.